scripts/release_scripts/update_configs.py

```diff
@@ -307,6 +307,39 @@ def add_mailchimp_api_key(release_feconf_path):
             f.write(line)
 
 
+def add_release_tester_email_ids(release_feconf_path):
```

Comment:
Hmm, shouldn't we rather add a config variable for this? The problem I see with this approach is that we would need to redeploy if we need to enable new email addresses for more emails.

Reply:
Sorry, I did not understand this. We are adding a config variable to feconf but the value is being supplied by release co-ordinator. Do you mean a config variable on the server instead?

Reply:
Yeas, the configs that we can modify in the admin page.